<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveViewTargetItem</class>
 <widget class="QWidget" name="LiveViewTargetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="MyLabel" name="labelChannel">
        <property name="text">
         <string>Channel: CAM1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="myLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TargetImageWidget" name="widgetImage" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="myLabel_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="labelTime">
        <property name="text">
         <string>2019-08-20 09:12:12</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="pageVca">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="MyLabel" name="labelVcaEvent">
         <property name="text">
          <string>Event: Entrance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyLabel" name="labelVcaObject">
         <property name="text">
          <string>Detection Object: Human</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAnpr">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="MyLabel" name="labelLicense">
         <property name="text">
          <string>License Plate: 123456</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MyLabel" name="labelPlateType">
            <property name="text">
             <string>Plate Type: Black</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelPlateColor">
            <property name="text">
             <string>Plate Color: Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelVehicleType">
            <property name="text">
             <string>Vehicle Type: Bus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelVehicleColor">
            <property name="text">
             <string>Vehicle Color: Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelVehicleBrand">
            <property name="text">
             <string>Vehicle Brand</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelSpeed">
            <property name="text">
             <string>Speed: 60km/h</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyLabel" name="labelDirection">
            <property name="text">
             <string>Direction: Approach</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MyLabel" name="labelLicense_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFace">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="MyLabel" name="labelFaceDetection">
         <property name="text">
          <string>Event: Face Detection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetFaceInfo" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="ToolButtonFace" name="toolButtonHat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/resource.qrc">
              <normaloff>:/face/face/hats.png</normaloff>:/face/face/hats.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="ToolButtonFace" name="toolButtonMale">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Famale</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/resource.qrc">
              <normaloff>:/face/face/female.png</normaloff>:/face/face/female.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="ToolButtonFace" name="toolButtonFaceMask">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/resource.qrc">
              <normaloff>:/face/face/mask.png</normaloff>:/face/face/mask.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="ToolButtonFace" name="toolButtonGlasses">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/resource.qrc">
              <normaloff>:/face/face/glasses.png</normaloff>:/face/face/glasses.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ToolButtonFace" name="toolButtonAge">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Adult</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/resource.qrc">
              <normaloff>:/face/face/adult.png</normaloff>:/face/face/adult.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MyLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
  <customwidget>
   <class>TargetImageWidget</class>
   <extends>QWidget</extends>
   <header>TargetImageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolButtonFace</class>
   <extends>QToolButton</extends>
   <header>ToolButtonFace.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
